packages/api/src/middleware/acl/checkGoogleJwt.js

```diff
@@ -9,6 +9,18 @@ async function checkGoogleJwt(req, res, next) {
   }
   const token = authorization.replace('Bearer ', '');
 
+  // For implicit flow from @react-oauth useGoogleLogin() hook from the email invite view
+  if (req.body.isAccessToken) {
```

Comment:
Looking at this again I am wondering if this should just be separated into two functions -- one for login and one for access token. Then the isAccess token would not have to be created at all on the frontend and the routes would just point to their respective functions. What do you think?